libraries/botbuilder-ai/botbuilder/ai/qna/models/qnamaker_trace_info.py

```diff
@@ -6,6 +6,7 @@
 from botbuilder.schema import Activity
 from .metadata import Metadata
 from .query_result import QueryResult
+from .qna_request_context import QnARequestContext
 
 
 class QnAMakerTraceInfo:
```

Comment:
Whoops, looks like a typo got in from last PR. Could you please correct this string to say `trace` instead of `trice`?